otx/algorithms/detection/adapters/mmdet/hooks/det_class_probability_map_hook.py

```diff
@@ -29,15 +29,16 @@
 class DetClassProbabilityMapHook(BaseRecordingForwardHook):
     """Saliency map hook for object detection models."""
 
-    def __init__(self, module: torch.nn.Module) -> None:
-        super().__init__(module)
+    def __init__(self, module: torch.nn.Module, normalize=True, use_cls_softmax=True) -> None:
```

Comment:
```suggestion
    def __init__(self, module: torch.nn.Module, normalize: bool =True, use_cls_softmax: bool =True) -> None:
```
How about to add type hint?